src/Mobs/Monster.h

```diff
@@ -251,6 +254,8 @@ class cMonster :
 	int m_Age;
 	int m_AgingTimer;
 
+	bool m_LastTargetWasAPlayer;
```

Comment:
Please rename to `m_WasLastTargetAPlayer` and the getter to `WasLastTargetAPlayer()`